taxonomy-editor-frontend/src/pages/startproject/index.tsx

```diff
@@ -96,10 +97,19 @@ const StartProject = ({ clearNavBarLinks }) => {
 
         <div>
           <TextField
+            error={branchNameError}
+            helperText={branchNameError && "Special characters or white spaces are not allowed"}
```

Comment:
As far as I remember even upper case characters are not allowed in branch name. 

You should test and confirm though.

Reply:
does this means we restrict to a-z, 0-9 and _ ?, if so, maybe we could test for that, think it would be easier that way

Reply:
@Jagrutiti yeah, I tested , the caps is also not allowed, so added check for that ! thanks.